templates/security/livepatch.html

```diff
@@ -11,431 +16,495 @@
+            Livepatch on-prem is designed for complex Enterprise environments that follow their own rollout policy and remain in control of which machines will get updated and when. Livepatch on-prem regularly syncs with the Ubuntu Livepatch service and obtains the latest patches. It then deploys the livepatches gradually in as many stages as required.
           </p>
+          <div class="p-cta-block">
+            <a href="/security/livepatch/docs/on_prem">Learn more about Livepatch on-prem server&nbsp;&rsaquo;</a>
+          </div>
         </div>
       </div>
-    </div>
-  </section>
-
-  <section class="p-strip is-deep is-bordered">
-    <div class="row u-equal-height">
-      <div class="col-3 u-vertically-center u-align--center u-hide--medium u-hide--small">
-        {{ image(url="https://assets.ubuntu.com/v1/0326ac7d-dena-logo.png",
-                alt="Dena",
-                width="144",
-                height="59",
+      <div class="p-image-container--cinematic is-cover is-highlighted">
+        {{ image(url="https://assets.ubuntu.com/v1/2e977117-chart-2.png",
+                alt="",
+                width="3696",
+                height="1541",
                 hi_def=True,
-                loading="auto",) | safe
+                loading="lazy",
+                attrs={"class": "p-image-container__image"}) | safe
         }}
       </div>
-      <div class="col-8 col-start-large-5">
-        <blockquote class="p-pull-quote">
-          <p class="p-pull-quote__quote u-no-max-width">
-            Livepatch is like a dream come true, both from a technical and a business standpoint. Our Ubuntu systems now
-            rarely, or never, have to be rebooted. Service is continuous. That makes a big difference for user and customer
-            satisfaction and loyalty.
-          </p>
-          <cite class="p-pull-quote__citation">Masaaki Hirose, IT Platform Department, DeNA</cite>
-        </blockquote>
-      </div>
     </div>
   </section>
 
-  <section class="p-strip--light">
-    <div class="row u-equal-height">
-      <div class="col-5 u-hide--medium u-hide--small u-align--center">
-        {{ image(url="https://assets.ubuntu.com/v1/c4b290c8-Contact+us.svg",
-                alt="",
-                width="281",
-                height="200",
+  <section class="p-section">
+    {% call(slot) vf_quote_wrapper(
+      title_text="What our customers say",
+      quote_size="small",
+      quote_text="Livepatch is a perfect fit for our needs. There’s no other solution like it, and it’s highly cost-effective. Manually migrating virtual machines, applying kernel updates, and rebooting took an average of 32 hours per server. Multiplied by 80 servers, that was more than 2,500 hours of work",
+      citation_source_name_text="Shinya Tsunematsu",
+      citation_source_title_text="Senior Engineering",
+      citation_source_organisation_text="Lead of Tech Division, GMO Pepabo",
```

Comment:
```suggestion
      citation_source_organisation_text="GMO Pepabo",
```
I believe it is like this